docs/hub/models-gated.md

```diff
@@ -63,6 +63,7 @@ You can automate the approval of access requests by using the API. You must pass
 The base URL for the HTTP endpoints above is `https://huggingface.co`.
 
 Those endpoints are not officially supported in `huggingface_hub` or `huggingface.js` yet but [this code snippet](https://github.com/huggingface/huggingface_hub/issues/1535#issuecomment-1614693412) (in Python) might help you getting started.
+NEW! There's an [open PR](https://github.com/huggingface/huggingface_hub/pull/1905) in `huggingface_hub` to add official support from our Python library.
```

Comment:
This is merged now :)

Reply:
yep i know but @Wauplin is too fast 💨

Reply:
I'm fine with opening a new PR once it's officially released so that we don't block this doc PR in the meantime (it's a matter of a few days). So fine to keep it like this for now :)